feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go

```diff
@@ -184,11 +218,12 @@ func awaitTimeout(ctx context.Context, c *fluent.GRIBIClient, t testing.TB, time
 
 // testArgs holds the objects needed by a test case.
 type testArgs struct {
-	ctx context.Context
-	c   *fluent.GRIBIClient
-	dut *ondatra.DUTDevice
-	ate *ondatra.ATEDevice
-	top *ondatra.ATETopology
+	ctx    context.Context
+	c      *fluent.GRIBIClient
+	dut    *ondatra.DUTDevice
+	ate    *ondatra.ATEDevice
+	top    *ondatra.ATETopology
+	nhtype string
```

Comment:
Same comment as others, I think a better way is to use the same style as https://github.com/openconfig/featureprofiles/pull/1335/files, so we avoid this magic `nhType` string

cc: @bkreddy143